src/Tizen.System.Feedback/Feedback/Feedback.cs

```diff
@@ -316,26 +259,16 @@ public bool IsSupportedPattern(FeedbackType type, String pattern)
         /// </example>
         public void Play(FeedbackType type, String pattern)
         {
-            bool found = false;
-            int i = 0;
+            int number;
             Interop.Feedback.FeedbackError res;
 
-            for (i = 0; i < Constants.NumberOfPattern; i++)
-            {
-                if (String.Compare(pattern, Pattern[i].PatternString) == 0)
-                {
-                    found = true;
-                    break;
-                }
-            }
-
-            if (!found)
+            if (!Pattern.TryGetValue(pattern, out number))
                 throw new ArgumentException("Invalid Arguments");
```

Comment:
Provide move detail message and param name:
```C#
throw new ArgumentException($"Not supported pattern string : {pattern}", nameof(pattern));
```

Reply:
Done. :)